src/agent/debuglet.ts

```diff
@@ -522,15 +576,20 @@ export class Debuglet extends EventEmitter {
    * @param {number} seconds
    * @private
    */
-  scheduleBreakpointFetch_(seconds: number): void {
+  scheduleBreakpointFetch_(seconds: number, onlyOnce: boolean): void {
```

Comment:
Add JSDoc for `onlyOnce`. Also, naming suggestion -- `once` is just as descriptive as `onlyOnce`.

Reply:
Done.